src/sentry/utils/security/orgauthtoken_token.py

```diff
@@ -43,3 +43,7 @@ def parse_token(token: str):
 
 def base64_encode_str(str):
     return b64encode(str.encode("ascii")).decode("ascii")
+
+
+def hash_token(token: str):
+    return hashlib.sha256_text(token).hexdigest()
```

Comment:
Would this make more sense to have implemented on the `OrgAuthToken` model in the `create` function? With it implemented there, we would know that all tokens will be hashed. 

In the current state with separated into a util function, a developer would have to know to hash the token first and then create the entry. It's possible for us to have non-hashed and hashed tokens.

Reply:
The reason I extracted this out into a util is because we also need to hash in the same way when we look up the token in the DB - so having a util ensures it is hashed the same way on creation & on authentication?